CAP/gap/MethodRecord.gi

```diff
@@ -1054,6 +1054,13 @@ SubtractionForMorphisms := rec(
   end,
   return_type := "morphism" ),
 
+MultiplyWithElementOfCommutativeRingForMorphisms := rec(
+  installation_name := "MultiplyWithElementOfCommutativeRingForMorphisms",
+  filter_list := [ IsRingElement, "morphism" ],
```

Comment:
@sebasguts 
I think this is the first time where we have a filter that is not in:
IsList, object, morphism, IsInt
Can this cause problems?

Reply:
Moreover, a linear category over a commutative ring should
know over which commutative ring it is defined.
I suggest to declare an attribute like
`CommutativeRingOfLinearCategory`
for categories with the property
`IsLinearCategoryOverCommutativeRing`.

Having this ring available would enable us
to type-check the ring element in the pre-function
of the multiplication.

Reply:
I agree, in which file shall I do the declaration? `CAP.gd`?

Reply:
The filter does not provide any problem. You can set whatever filter you like there. However, only the keywords `"object"`, `"morphism"`, and `"twocell"` are replaced with the specific category filters.